poll/poll.py

```diff
@@ -810,7 +810,9 @@ def generate_report_data(self, user_state_iterator, limit_responses=None):
                 # End the iterator here
                 return
 
-            choice = user_state.state['choice']  # {u'submissions_count': 1, u'choice': u'R'}
+            choice = user_state.state.get('choice')  # {u'submissions_count': 1, u'choice': u'R'}
+            if choice is None:
+                continue
```

Comment:
I'm not sure this is a valid scenario since all fields of a block are saved together from what I understand.

Reply:
I've left this check in, as a precaution, and added a test for where the value is `None`.